gulpfile.js

```diff
@@ -101,12 +101,18 @@ function test() {
         log("\nTesting cases that don't match pattern: extensionContext");
     }
 
+    const testResultsPath = path.join(__dirname, "test", "DebuggerTests.xml");
+    process.env.MOCHA_FILE = testResultsPath;
     return gulp.src(["test/**/*.test.js", "!test/extension/**"])
         .pipe(mocha({
             ui: "tdd",
             useColors: true,
             invert: !options.pattern,
-            grep: options.pattern || "(extensionContext|localizationContext)"
+            grep: options.pattern || "(extensionContext|localizationContext)",
+            reporter: "mocha-multi-reporters",
+            reporterOptions: {
+                configFile: path.resolve("test/smoke/mochaReporterConfig.json"),
```

Comment:
I'd suggest to move `test/smoke/mochaReporterConfig.json` to `test/mochaReporterConfig.json` as it's global test config